packages/api-client/schema/index.ts

```diff
@@ -798,7 +798,7 @@ export default {
             required: false,
             schema: {
               title: 'Limit',
-              maximum: 1000.0,
+              maximum: 5000.0,
```

Comment:
iirc this is generated, you should update the source instead.

Reply:
yeah I regenerated the `api-client` and this was automatically populated, I didn't modify it myself